knowledge-base/drawer-chart-refresh.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Refresh a Chart Inside a Drawer
+description: How to refresh a Chart within a Drawer content.
+type: how-to
+page_title: Refresh a Chart Inside a Drawer
+slug: drawer-kb-chart-refresh
+position: 
+tags: 
+ticketid: 1583102
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Drawer for Blazor,<br /> Charts for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+This knowledge base article answers the following questions:
+
+* How do I refresh a Chart inside a Drawer?
+* Is it possible to refresh a Chart when the Drawer is expanded?
+
+## Solution
+
+1. Create a bool flag and set it within the Drawer's [`ExpandedChanged`]({%slug drawer-events%}#expandedchanged) handler to indicate when the Chart needs to be refreshed
+2. Call the Chart's [`Refresh()` method]({%slug components/chart/overview%}#chart-reference-and-methods) within the `OnAfterRenderAsync` lifecycle method after a brief delay that allows the Drawer to finish expanding
```

Comment:
Personally, I would split the sentence in two for conciseness and clarity. For example:

```suggestion
2. Call the Chart's [`Refresh()` method]({%slug components/chart/overview%}#chart-reference-and-methods) within the `OnAfterRenderAsync` lifecycle method. Set a brief delay to allow the Drawer to finish expanding before the refresh.
```